backend/app/processing/free_llm_formatter.py

```
@@ -0,0 +1,80 @@
+    }
+    
+    payload = {
+        "inputs": prompt,
+        "parameters": {
+            "max_length": 102400,
+            "temperature": 0.7,
+            "top_p": 0.9
+        }
+    }
+    
+    response = requests.post(
```

Ruff (0.8.2)

Probable use of `requests` call without timeout

(S113)